### DISCUSSION TITLE
SCTransform without integration

### DISCUSSION THREAD

**Discussion by Chengwei94** [satijalab/seurat#4862] [Q&A]
If I merge 3 different datasets using the merge function(no integration), should I be doing SCTtransform before or after merging? This question is mainly for the DE analysis since that if I do integration, I would be conducting DE analysis on data which are SCTransformed individually before integrated. 

Follow up question: Wouldnt wilcoxon test be very sensitive to batch effects, since it can be interpreted as testing the median of between 2 distributions. If so, whats the suggested test for DE analysis with datasets that are heavily batched?

**Comment by saketkc** [comment]
I answered this in a related question here: https://github.com/satijalab/seurat/issues/5009#issuecomment-907426252
